src/Tizen.NUI/src/internal/ComponentApplication.cs

```diff
@@ -152,31 +150,19 @@ internal ApplicationSignal CreateNativeSignal()
             if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
             return ret;
         }
-		
 
-		
-		// Callback for Application InitSignal
-		private IntPtr OnApplicationCreateNative(IntPtr data)
-        {
-            IntPtr ptr = IntPtr.Zero;
-		
-			if (_applicationCreateNativeEventHandler != null)
-			{
-				ptr = _applicationCreateNativeEventHandler.Invoke(data);
-			}
-		    return ptr;
-		}
-
-        internal void RegisterFrameComponent(NUIFrameComponent nuiFrameComponent)
+        // Callback for Application InitSignal
+        private IntPtr OnApplicationCreateNative()
         {
-            _frameComponentList.Add(nuiFrameComponent);
-        }
+            IntPtr handle = IntPtr.Zero;
 
-        internal int GetFrameComponentCount()
-        {
-            return _frameComponentList.Count;
-        }
+            if (_applicationCreateNativeEventHandler != null)
```

Comment:
I think this can be simplified like this? how do you think?

handle = _applicationCreateNativeEventHandler?.Invoke();

Reply:
handle = _applicationCreateNativeEventHandler?.Invoke();
->
Error	CS0266	Cannot implicitly convert type 'System.IntPtr?' to 'System.IntPtr'. An explicit conversion exists (are you missing a cast?)